README.md

```diff
@@ -2,12 +2,15 @@
 
 This is an implementation of a configurable dropdown for use with the Elm UI package.
 
-It tries to follow the advice from 
+![preview](preview.png)
+
+https://package.elm-lang.org/packages/PaackEng/elm-ui-dropdown/1.1.0/
```

Comment:
I don't think this link is a great idea:
1 - We would have to keep updating it;
2 - It would appear on that page, cause elm-lang reuses the repo's README;
3 - It's not actually related to this PR.

Instead, we could put this link in the repo's description in GitHub

Reply:
I asked the link to be put in the description, but I was ignored 🤷🏻‍♂️  I think it is important though since I did not even know that this package was published! ☝🏻

Reply:
hey @kutyel sorry again about that, because of a configuration setup in the org account, it seems that no one was watching this repo. but I will add it in the description